platform/ios/src/MGLMapView.h

```diff
@@ -493,6 +493,19 @@ MGL_EXPORT IB_DESIGNABLE
  */
 @property(nonatomic, getter=isPitchEnabled) BOOL pitchEnabled;
 
+/**
+ A Boolean value that determines whether the user will receive haptic feedback
+ for certain interactions with the map.
```

Comment:
Going forward, any gesture that results in a snapping motion should trigger haptic feedback, if only for consistency. What if the developer returns `NO` in `-[MGLMapViewDelegate mapView:shouldChangeFromCamera:toCamera:]`?